xml/System/DateTimeOffset.xml

```diff
@@ -1079,10 +1176,29 @@
         <Parameter Name="microseconds" Type="System.Double" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="microseconds">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="microseconds">A number of whole and fractional microseconds. The number can be negative or positive.</param>
+        <summary>Returns a new <see cref="T:System.DateTimeOffset" /> object that adds a specified number of microseconds to the value of this instance.</summary>
+        <returns>An object whose value is the sum of the date and time represented by the current <see cref="T:System.DateTimeOffset" /> object and the number of whole microseconds represented by <paramref name="microseconds" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The fractional part of value is the fractional part of a microsecond.
+             For example, 4.5 is equivalent to 4 microseconds and 50 ticks, where one microseconds = 10 ticks.
+             However, `microseconds` is rounded to the nearest microsecond; all values of .5 or greater are rounded up.
+            
+Because a <xref:System.DateTimeOffset> object does not represent the date and time in a specific time zone,
+             the <xref:System.DateTimeOffset.AddMicroseconds(System.Double)> method does not consider a particular time zone's adjustment rules
+             when it performs date and time arithmetic.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">The resulting <see cref="T:System.DateTimeOffset" /> value is less than <see cref="F:System.DateTimeOffset.MinValue" />
```

Comment:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">The resulting <see cref="T:System.DateTimeOffset" /> value is less than <see cref="F:System.DateTimeOffset.MinValue">DateTimeOffset.MinValue</see>.
```

Reply:
This improves the link text since otherwise it would just show "MinValue" - and there are lots of MinValues.